packages/sources/armanino/src/transport/mco2.ts

```diff
@@ -0,0 +1,42 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { BaseEndpointTypes } from '../endpoint/mco2'
+
+export interface ResponseSchema {
+  totalMCO2: number
+  totalCarbonCredits: number
+  timestamp: string
+}
+
+export type HttpTransportTypes = BaseEndpointTypes & {
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+export const httpTransport = new HttpTransport<HttpTransportTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      return {
+        params: [param],
+        request: {
+          baseURL: config.API_ENDPOINT,
+          url: 'MCO2',
+        },
+      }
+    })
```

Comment:
Since there's only one request this endpoint can have maybe we can do something like this

```suggestion
    return {
        params,
        request: {
          baseURL: config.API_ENDPOINT,
          url: 'MCO2',
        },
     }
```

Reply:
Just noticed you already did this for STBT so just the same as that here.